topic-configuration/README.adoc

```diff
@@ -0,0 +1,689 @@
+////
+START GENERATED ATTRIBUTES
+WARNING: This content is generated by running npm --prefix .build run generate:attributes
+////
+
+
+:community:
+:imagesdir: ./images
+:product-version: 1
+:product-long: Application Services
+:product: App Services
+// Placeholder URL, when we get a HOST UI for the service we can put it here properly
+:service-url: https://cloud.redhat.com/beta/application-services/streams/
+:property-file-name: app-services.properties
+
+// Other upstream project names
+:samples-git-repo: https://github.com/redhat-developer/app-services-guides
+
+//URL components for cross refs
+:base-url: https://github.com/redhat-developer/app-services-guides/blob/main/
+:base-url-cli: https://github.com/redhat-developer/app-services-cli/tree/main/docs/
+:getting-started-url: getting-started/README.adoc
+:kafka-bin-scripts-url: kafka-bin-scripts/README.adoc
+:kafkacat-url: kafkacat/README.adoc
+:quarkus-url: quarkus/README.adoc
+:rhoas-cli-url: rhoas-cli/README.adoc
+:rhoas-cli-ref-url: commands
+:topic-config-url: topic-configuration/README.adoc
+
+////
+END GENERATED ATTRIBUTES
+////
+
+[id="chap-configuring-topics"]
+= Configuring topics in {product-long}
+ifdef::context[:parent-context: {context}]
+:context: configuring-topics
+
+[IMPORTANT]
+====
+{product-long} is currently available for Development Preview. Development Preview releases provide early access to a limited set of features that might not be fully tested and that might change in the final GA version. Users should not use Development Preview software in production or for business-critical workloads. Limited documentation is available for Development Preview releases and is typically focused on fundamental user goals.
+====
+
+// Purpose statement for the assembly
+[role="_abstract"]
+As a developer of applications and services, you can refer to the properties of your topics in {product} to better understand how the Kafka implementation is managed for your services. You can edit certain topic properties according to the needs and goals of your services.
```

Comment:
Does this need to be mentioned? _As a developer of applications and services_. Will this vary according to organisation?

Reply:
Well, it can vary by story. This is just the suggested user-story statement format that I've always been told to user for assembly intros, i.e., "As a ___, I want to ____ so that ____" Makes it immediately clear what user story this assembly addresses, including target user persona, objective, and reason.

Reply:
Okay